storage/badger/dkg_state.go

```diff
@@ -0,0 +1,144 @@
+package badger
```

Comment:
Can you add godoc comments for the exported funcs

Reply:
https://github.com/onflow/flow-go/pull/1720/commits/23626c59c4dc4a91535a3b3a6c81476b442c6c4d